documentation/src/main/asciidoc/stories/assembly_partition_handling.adoc

```diff
@@ -1,8 +1,23 @@
 [id='partition-handling']
 :context: partition-handling
-= Setting up partition handling
+= Configuring {brandname} to handle network partitions
```

Comment:
Should this whole section also be linked in "Planning and Tuning Infinispan Deployments" guide?

Reply:
I think it would be too much for the "Planning and Tuning" guide honestly. I think that content is suitable for explaining CAP and the trade-offs plus the warning about avoiding partitions with GC pauses. It is tempting to add more to the planning guide but I feel like we've got a good balance at the moment.